ga4gh/cli.py

```diff
@@ -109,12 +109,23 @@ def _run(self, methodName, attrName=None):
         method = getattr(self._httpClient, methodName)
         results = method(self._request)
         for result in results:
-            if attrName is None:
-                print(result)
+            if self._verbosity == 1:
+                print(methodName, "request: \n", self._request)
```

Comment:
It would probably be better to prefer using string formatting (using str's `format` method) over calling print with varargs